### PR TITLE
[CF-30]: Build UI home page (new design)

### PR DESCRIPTION
**Pull request by Tam2690**

### CODE REVIEW

File: src/components/HomePageCoffeeShop/styles.js [hunk 1]
```diff
@@ -1,10 +1,19 @@
 import styled from 'styled-components';
 
 const HomePageCoffeeShopStyled = styled.div`
+  --primary-padding: 0 10px;
+  --primary-border: 2px solid #80808070;
```
Comment (thienphamIT1907) [inline src/components/HomePageCoffeeShop/styles.js]: Check your hex color (#rrggbb)

File: src/components/HomePageCoffeeShop/styles.js [hunk 2]
```diff
@@ -24,17 +33,38 @@ const HomePageCoffeeShopStyled = styled.div`
...
+    align-items: center;
+    padding: var(--primary-padding);
+
+    .like-btn {
+      font-size: 22px;
+      margin-right: 5px;
+    }
+
+    .favorite-btn {
+      font-size: 22px;
```
Comment (thienphamIT1907) [inline src/components/HomePageCoffeeShop/styles.js]: Use css variable

**Comment by thienphamIT1907** [comment]
@Tam2690 you need to style for image keep its aspect ratio

image

**Comment by thienphamIT1907** [comment]
Follow resolutions for responsive layout 

image

File: src/components/HomePageCoffeeShop/styles.js (hunk 1 above)
Comment (Tam2690) [reply src/components/HomePageCoffeeShop/styles.js]: origin hex code is #808080, then I reduce transparency on devtool and it returns #80808070

**Comment by Tam2690** [comment]
> @Tam2690 you need to style for image keep its aspect ratio
> 
> image

File: src/components/HomePageHeaderNavbar/styles.js [hunk 3]
```diff
@@ -38,31 +38,23 @@ export const HomePageHeaderNavbarStyled = styled.div`
...
+        font-weight: bold;
         font-size: 16px;
       }
     }
   }
 `;
 
 export const LogoStyled = styled.img`
-  width: 80px;
   height: 80px;
```
Comment (thienphamIT1907) [inline src/components/HomePageHeaderNavbar/styles.js:56]: You need to set width and height, object-fit property also for keep aspect ratio of this image

File: src/components/HomePageCoffeeShopsList/index.jsx [hunk 4]
```diff
@@ -1,18 +1,44 @@
...
-    <HomePageCoffeeShop />
-    <HomePageCoffeeShop />
-    <HomePageCoffeeShop />
-    <HomePageCoffeeShop />
-    <HomePageCoffeeShop />
-    <HomePageCoffeeShop />
-    <HomePageCoffeeShop />
-    <HomePageCoffeeShop />
+    <Row gutter={[32, 32]}>
+      <Col xxl={{ span: 6 }} xl={{ span: 8 }} lg={{ span: 12 }}>
```
Comment (thienphamIT1907) [inline src/components/HomePageCoffeeShopsList/index.jsx:8]: You should use `<Col xxl={6} xl={8} lg={12}>` later in other task. This is not critical

**Review by thienphamIT1907 — APPROVED** [review]

**Pull request merged**